examples/00-mapdl-examples/composite_dcb.py

```diff
@@ -0,0 +1,415 @@
+"""
+.. _ref_dcb_test_composite_delamination
+
+======================================================================
+Static simulation of double cantilever beam test via cohesive elements
+======================================================================
+
+This example is inspired to a classic double cantilever beam test commonly used
+to study mode I interfacial delamination of composite plates.
+
+Description
+===========
+
+Objective
+~~~~~~~~~
+
+This example shows how to use PyMAPDL to simulate delamination in
+composite materials. PyDPF modules are also used for the post-processing of the results.
+
+Problem Figure
+~~~~~~~~~~~~~~
+.. image:: ../../../images/dcb_test.png
+   :width: 400
+   :alt: DCB experimental test setup
+
+Procedure
+~~~~~~~~~
+* Launch MAPDL instance
+* Setup the model
+* Solve the model
+* Plot results of interest using PyMAPDL
+* Plot results using PyDPF
+* Plot reaction force
+
+Additional Packages Used
+~~~~~~~~~~~~~~~~~~~~~~~~
+* `Matplotlib <https://matplotlib.org>`_ is used for plotting purposes.
+* `Pandas <https://pandas.pydata.org/>`.
+"""
+
+###############################################################################
+# Starting MAPDL as a service
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# This example begins by launching Ansys Mechanical APDL.
+
+import os
+
+from ansys.dpf import core as dpf
+from ansys.dpf.core import Model
+import numpy as np
+import pyvista as pv
+import matplotlib.pyplot as plt
+
+from ansys.mapdl import core as pymapdl
+
+# start MAPDL as a service
+mapdl = pymapdl.launch_mapdl()
+print(mapdl)
+
+###############################################################################
+# Setting the model geometrical inputs
+# ========================
+length = 75.0
+pre_crack = 10.0
+width = 25.0
+height = 1.7
+d = 10.0
+# a small quantity defined for avoiding rounding-off error when picking geometrical entities
+eps = 1e-1 
+
+###############################################################################
+# Setting up the model
+# ====================
+# In this step the model setup is performed choosing the units system and the
+# element types to be used in the simulations. In this case a fully 3D approach
+# is chosen therefore ``SOLID186`` elements are used for meshing volumes and
+# ``TARGE170``/``CONTA174`` are used for modelling cohesive elements in between contact
+# surfaces.
```

Comment:
```suggestion
# Set up the model
# ================
# Set up the model by choosing the units system and the
# element types for the simulations. Because a fully 3D approach
# is chosen for this example, ``SOLID186`` elements are used for meshing volumes, and
# ``TARGE170`` and ``CONTA174`` are used for modelling cohesive elements in between contact
# surfaces.
```